django/common/scripts/cleaning/clean_long_text_ORBIS.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from common.scripts import utils
+
+def clean_long_text_ORBIS(raw_input):
+    """Remove special charater from long BLOB / CLOB text in ORBIS"""
+    if utils.is_empty(raw_input):
+        return None
+
+    text = (
+        raw_input.replace("\x01", "")
+        .replace("\x00", "")
+        .replace("\x1a", "")
+        .replace("\x03", "")
+        .replace("\x0c", "")
+        .replace("'", "")
+        .replace("\r", "\n")
+        .replace("\\m0", "")
+        .replace("\\m1", "")
+        .replace("\\m2", "")
+        .replace("\\m3", "")
+        .replace("\\m8", "")
+        .replace("\\\\", "")
+        .replace("\\f", "\\\f")
+    )
+
+    if "\f:20" in text:
+        x = re.findall("(.*)(?=\\f:[\\d]{4})", text)
+        y = re.findall("\\\f:[\\d]{4}\\\\?", text)
+        text = text.replace(x[0], "").replace(y[0], "")
+
+    elif "\f:12" in text:
+        x = re.findall("(.*)(?=\\f:[\\d]{5})", text)
+        y = re.findall("\\\f:[\\d]{5}\\\\?", text)
+        text = text.replace(x[0], "").replace(y[0], "")
+
+    else :
```

Comment:
Not sure you want this `else`, do you? It wasn't here in the previous version

Reply:
If the beginning of the document contains \ f:20 or \ f:12, we must remove everything that is before while in the other documents, you must remove only the first two characters if at least one of them is not a letter. There are examples in the test_clean_long_text_ORBIS file of the text beginnings that I found. So there is two different type of documents : 
1/ contains \f:20 or 12
2/ begins with two chacarters (at least one of them is not a letter)

Reply:
Other details: but before, there was no space before your `:` but there were spaces before and after `=`, precommits including black aren't setup on fhir-river ?